app/views/editor/_main_image.html.erb

```diff
@@ -18,7 +18,7 @@
       </span>
 
       <!-- http://stackoverflow.com/questions/11235206/twitter-bootstrap-form-file-element-upload-button -->
-      <label class="" for="side-fileinput">
+      <label class="cursor-dropzone" for="side-fileinput">
```

Comment:
```suggestion
      <label style="cursor:pointer" for="side-fileinput">
```
The CSS can be directly applied here. So that you just need to change only one line of code :smile: 
There is no need to modify the CSS file. 
Thanks!

Reply:
I am fine with both. You want to accept this suggestion @ValeriaCobianISW?

Reply:
Sure, thanks @gauravano